nrf52840_layout.ld

```diff
@@ -0,0 +1,17 @@
+/* Layout for the nRF52840-DK, usable by the examples in this repository. */
+
+MEMORY {
+  /* The application region is 64 bytes (0x40) */
+  FLASH (rx) : ORIGIN = 0x00030040, LENGTH = 0x0005FFC0
```

Comment:
If I remember correctly, there is 1MB of flash, so the length should be `0x000CFFC0`

Reply:
That is correct.